translations/monero-core_de.ts

```diff
@@ -2129,15 +2137,15 @@ Für einen reinen Sendenachweis muss die Empfängeradresse nicht angegeben werde
     <message>
         <location filename="../wizard/WizardCreateWalletFromDevice.qml" line="112"/>
         <source>Create a new wallet from hardware device</source>
-        <translation type="unfinished"></translation>
+        <translation>Erstelle eine neue Wallet von einem Hardware-Gerät</translation>
```

Comment:
s.o. `ein neues Wallet`

Reply:
Die Brieftasche, die Geldbörse, die (Brief)Tasche

But on the other hand just
Das Portemonnaie (which is not an original german word and less common)